File: 8kyu/Total amount of points/GO/v1/kata.go
package kata

import (
	"strconv"
	"strings"
)

func Points(games []string) int {
	totalPointsA := 0
	for _, game := range games {
		score := strings.Split(game, ":")
		scoreA, _ := strconv.Atoi(score[0])
		scoreB, _ := strconv.Atoi(score[1])

		if scoreA > scoreB {
			totalPointsA += 3
		} else if scoreA == scoreB {
			totalPointsA++
		}
	}

	return totalPointsA
}

File: 8kyu/Total amount of points/C#/v1/KataSolution/Kata.cs
namespace KataSolution;

public static class Kata {
    public static int TotalPoints(string[] games)
    {
        int totalPointsA = 0;
        foreach (var game in games)
        {
            var score = game.Split(":");
            var scoreA = int.Parse(score[0]);
            var scoreB = int.Parse(score[1]);

            if (scoreA > scoreB) totalPointsA += 3;
            else if (scoreA == scoreB) totalPointsA++;
        }

        return totalPointsA;
    }
}